private/performOptimizationCs.m

```diff
@@ -0,0 +1,59 @@
+function parameters = performOptimizationCs(parameters, negLogPost, iMS, par0, J_0, options)
```

Comment:
what does C stand for in the function name?

Reply:
coordinate search, one might go for the long name here

Reply:
yes, please change it here